src/osd/PeeringState.cc

```diff
@@ -98,7 +98,8 @@ void PGPool::update(OSDMapRef map)
 
 /*-------------Peering State Helpers----------------*/
 #undef dout_prefix
-#define dout_prefix (dpp->gen_prefix(*_dout))
+#define dout_prefix (dpp->gen_prefix(*_dout)) \
+        << "PeeringState::"
```

Comment:
Hmm, good idea.  How about "PeeringState::" <<  __func__  and remove the __func__ from elsewhere in this file?

Reply:
Note, that won't behave correctly for any uses withing lambdas, so do check for that.

Reply:
Updated.